### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ashish1595**

### CODE REVIEW

File: src/main/java/com/incentives/piggyback/notification/utils/Constant.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.incentives.piggyback.notification.utils;
+
+public interface Constant {
+	Integer SUCCESS_STATUS = 200;
+	Integer FAILURE_STATUS = 101;
+
+	String LOCATION_PUBLISHER_TOPIC = "locationEvents";
+	String LOCATION_SOURCE_ID = "1";
+
```
Comment (piggy1-mvn) [inline src/main/java/com/incentives/piggyback/notification/utils/Constant.java]: Should be using Notification constants here.

Comment (rajan123456) [inline src/main/java/com/incentives/piggyback/notification/utils/Constant.java]: Should be using notification constants here.

File: .classpath [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rajan123456) [inline .classpath:1]: Should add .classpath and .settings files to .gitignore

File: src/main/java/com/incentives/piggyback/notification/controller/NotificationController.java [hunk 3]
```diff
@@ -1,21 +1,42 @@
...
+	@Autowired
+	private NotificationService notificationService;
+	
 
     @Autowired
     private NotificationEventPublisher.PubsubOutboundGateway messagingGateway;
 
+	@RequestMapping(value = "/broadcast", method = RequestMethod.POST)
+	public ResponseEntity<RestResponse<String>> broadcastNotification(@RequestBody 
+			BroadcastRequest broadcastRequest) throws PiggyException {
```
Comment (rajan123456) [inline src/main/java/com/incentives/piggyback/notification/controller/NotificationController.java:36]: Should push notification created event to pub/sub. Added a new story to take care of that.

**Review by rajan123456 — APPROVED** [review]

**Pull request merged**